stripe/src/main/java/com/stripe/android/model/StripeJsonUtils.java

```diff
@@ -547,7 +548,7 @@ public static void putObjectIfNotNull(
     }
 
     @Nullable
-    public static String nullIfNullOrEmpty(@Nullable String possibleNull) {
+    static String nullIfNullOrEmpty(@Nullable String possibleNull) {
```

Comment:
I can't believe the language translation sometimes literally adds the string "null" as a value.